src/app/workflow/executions/executions-tab.component.html

```diff
@@ -1,6 +1,9 @@
 <div *ngIf="!metricsExist" class="w-100">
   <mat-card appearance="outlined" class="alert alert-info mt-3" role="alert" data-cy="no-metrics-banner">
-    <mat-card-content> <mat-icon class="alert-info-icon">info</mat-icon> No metrics data exist for this version. </mat-card-content>
+    <mat-card-content>
+      <mat-icon class="alert-info-icon">info</mat-icon> No metrics data exist for this
+      {{ version ? 'version' : entry?.entryTypeMetadata?.term }}.
```

Comment:
Was like this already, but just skip the period 
https://m3.material.io/foundations/content-design/style-guide/grammar-and-punctuation